content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/linux.md

```diff
@@ -52,6 +52,24 @@ Available versions are listed in source repositories for each language:
 - [Ruby][12] (`ruby`)
 - [PHP][13] (`php`)
 
+## Configure Unified Service Tags
+
+Unified Service Tags (USTs) apply consistent tags across traces, metrics, and logs, making it easier to navigate and correlate your observability data. Learn how to [set USTs for Linux services][16].
+
+## Enable SDK-dependent products and features
+
+After SSI loads the Datadog SDK into your applications and enables distributed tracing, you can configure additional products that rely on the SDK. These include capabilities such as Continuous Profiler, Application Security Monitoring, and trace ingestion controls.
+
+Use one of the following setup methods:
+
+- **[Configure in `application_monitoring.yaml`][18]**:
+
+  Enable products across all services on a host without modifying application command lines.
+
+- **[Set environment variables][17]**:
```

Comment:
```suggestion
- **[Set environment variables][17]**:
```
ok this is the 3rd one so now i'm assuming this IS the correct link LOL! I was just expecting it to take me to something more specific with envars